NAMESPACE

```diff
@@ -160,6 +162,7 @@ export(extract_argument)
 export(extract_frosting)
 export(extract_layers)
 export(extrapolate_quantiles)
+export(filter)
```

Comment:
musing: this does introduce a conflict with stats::filter, but I guess eventually we should be providing our own convolution utilities.

Reply:
Yeah, that will happen anytime dplyr is used. This would have happened before anyway since epiprocess was in Depends and it exports `filter()`.

Reply:
Hmm... I'm not sure why we export filter from epiprocess; shouldn't be necessary just to provide an S3 method.

Reply:
We could make this happen only when user attaches dplyr themselves.  I guess if users are using it a lot though then it might make sense to re-export.  It's just that convolution does seem somewhat popular with potential users.